core/frontend/src/RealityDataSource.ts

```diff
@@ -287,8 +287,10 @@ class RealityDataSourceImpl implements RealityDataSource {
       this._tilesetUrl = this.key.id;
     } else if (this.key.provider === RealityDataProvider.CesiumIonAsset ) {
       this._tilesetUrl = this.key.id;
-      if (this.key.id === CesiumIonAssetProvider.osmBuildingId)
+      if (this.key.id === CesiumIonAssetProvider.osmBuildingId) {
         this._tilesetUrl = getCesiumOSMBuildingsUrl();
+        this._isUrlResolved=true;
```

Comment:
Did you run lint? I would have expected it to complain about lack of whitespace around `=`.

Reply:
Lint didn't catch this, change to add whitespace around =